pkgs/applications/audio/ardour/default.nix

```diff
@@ -12,17 +12,17 @@
 , fftw
 , fftwSinglePrec
 , flac
-, fluidsynth
+# , fluidsynth
```

Comment:
Please add a comment explaining when these should be added back in or remove entirely.

Reply:
I removed them entirely, as they are unneeded.